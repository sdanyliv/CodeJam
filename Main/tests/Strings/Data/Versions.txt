1.001
1.2
1.002
1.02
1.01
1.09
1.101
1.102
1.010
1.10
1.200
1.199
1.198
1.1
1.0.0-alpha1
1.0.0-beta1
1.0.0-beta2
1.0.0-beta12
1.0.0-beta
1.0.0-beta.1
1.0.0-beta.12
1.0.0