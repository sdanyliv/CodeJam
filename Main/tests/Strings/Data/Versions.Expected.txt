1.0.0
1.0.0-alpha1
1.0.0-beta
1.0.0-beta.1
1.0.0-beta.12
1.0.0-beta1
1.0.0-beta2
1.0.0-beta12
1.1
1.01
1.001
1.2
1.02
1.002
1.09
1.10
1.010
1.101
1.102
1.198
1.199
1.200